en/news/_posts/2019-04-23-move-to-git-from-svn.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: news_post
+title: "Move to Git from Subversion"
+author: "hsbt"
+translator:
+date: 2019-04-23 00:00:00 +0000
+lang: en
+---
+
+Today, The canonical repository of the Ruby language was moved to Git from Subversion.
+
+The new repository is https://git.ruby-lang.org built by cgit. We can keep your commit hash on the ruby repository directly.
+
+## Development policy
+
+* We don't use a topic branch on cgit until Ruby 2.7 is released. ruby_2_7 will use cgit.
```

Comment:
Oops sorry, I misunderstood the original sentence by myself (topic branch is completely different from stable branch...). Let's rollback that part and note about ruby_2_7 in the later sentence...

```suggestion
* We don't use a topic branch on cgit.
```